src/shared/Table/TableView.stories.tsx

```diff
@@ -102,6 +102,8 @@ Example.args = {
 export const KebabAndButtonToolbarVisible = Template.bind({});
 KebabAndButtonToolbarVisible.args = {
   data: sampleData,
+  toolbarKebabAriaLabel: "kebab-dropdown-toolbar",
+  toolbarAriaLabel: "table-toolbar",
   kebabActions: [
```

Comment:
Aria labels are user-facing contents and are not intended to provide IDs on HTML elements. These need to be human-readable strings.